conclave-common/src/main/kotlin/com/r3/conclave/common/internal/PluginUtils.kt

```diff
@@ -9,7 +13,23 @@ object PluginUtils {
     const val GRAMINE_MANIFEST = "java.manifest"
     const val GRAMINE_SGX_MANIFEST = "java.manifest.sgx"
     const val GRAMINE_SGX_TOKEN = "java.token"
+    const val GRAMINE_SECCOMP = "gramine-seccomp.json"
```

Comment:
This is not used by the plugin I don't think.

Reply:
Correct, it was a leftover. I will move this in `GramineEnclaveHandle` were it is actually used.